Assets/Scripts/PlayerShooting.cs

```diff
@@ -4,25 +4,43 @@
 
 public class PlayerShooting : MonoBehaviour
 {
+    
     public GameObject arrowPrefab;
-    public float speed = 100f;
+    
+    public static float minForce = 0f;
+    public static float maxForce = 20f; 
+    public static float timeToGetToMaxForce = 2.5f;
+
+    private float shootingStrength = minForce;
+    private float timer = 0f;
 
     // Start is called before the first frame update
     void Start()
     {
-     
+        
     }
 
     // Update is called once per frame
     void Update()
     {
-        if (Input.GetMouseButtonDown(0))
+        if(Input.GetMouseButton(0))
+        {
+            timer += Time.deltaTime;
+            timer = Mathf.Clamp(timer, minForce, timeToGetToMaxForce);
+            shootingStrength = maxForce * timer;
```

Comment:
i dont understand this. if you multiply maxForce and timer and timer is greater than one, the force will be greater than the maxForce.

You should use a lerp. Something along the lines of 
`shootingStrength = Mathf.Lerp(minForce, maxForce, timer/timeToGetToMaxForce)`
"timer/timeToGetToMaxForce" should be a number between 0 and 1, which is how lerps are used. think of it as a percentage.